doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
+|:------------------------------------------------|:------------------------------------------------------------|:-------------------|
+|[icehello](./examples/icehello/)                 | How to send data to another process                         | :star:             |
```

Comment:
This example is missing. Therefore I would wait with a merge of this document until the examples are available or we remove those examples which are not present for now and add them later.

Reply:
It's prepared on another branch. I would like to agree on the list of examples for the Almond release and then implement them.